test/test_manifest.json

```diff
@@ -10219,6 +10219,17 @@
     "lastPage": 5,
     "type": "eq"
   },
+  {
+    "id": "issue17061_nowasm",
+    "file": "pdfs/issue17061.pdf",
+    "md5": "21a1b887b7e657b01e4205b22d1f4378",
+    "rounds": 1,
+    "link": true,
+    "noWasm": true,
+    "firstPage": 1,
+    "lastPage": 5,
+    "type": "eq"
+  },
```

Comment:
Could we perhaps use a smaller/simpler test-case instead, rather than one with five pages?
The following is probably one of the simplest/smallest JPEG2000 test-cases we have:
https://github.com/mozilla/pdf.js/blob/e3ea92603d17595aad4b1c37c7113cb7dee0c90e/test/test_manifest.json#L6407-L6421